cmd/clusterctl/client/config/providers_client.go

```diff
@@ -44,11 +44,13 @@ const (
 	// Bootstrap providers
 	KubeadmBootstrapProviderName = "kubeadm"
 	TalosBootstrapProviderName   = "talos"
-	EKSBootstrapProviderName     = "eks"
+	EKSBootstrapProviderName     = "eks" // DEPRECATED: if this provider is used it returns the same as aws-eks
```

Comment:
Can we remove this altogether?

Reply:
I added it back in due to the apidiff errors. Removing now.